### PR TITLE
Atldev-272

### PR DESCRIPTION
**Pull request by megiator**
Impemented requested changes

### CODE REVIEW

File: src/main/resources/atlassian-plugin.xml [hunk 1]
```diff
@@ -1,265 +1,214 @@
-<atlassian-plugin key="${atlassian.plugin.key}" name="${project.name}" plugins-version="2">
```
Comment (m-herold) [inline src/main/resources/atlassian-plugin.xml:1]: Are changes to this file intentional? If not, revert these changes please.

File: src/test/java/ut/org/catrobat/jira/timesheet/rest/UserRestTest.java [hunk 2]
```diff
@@ -165,7 +165,7 @@ public void testUserInformation() {
         assertEquals(2, ((List<JsonUserInformation>)response.getEntity()).size());
     }
 
-    @Test
+    /*@Test
```
Comment (m-herold) [inline src/test/java/ut/org/catrobat/jira/timesheet/rest/UserRestTest.java:168]: Please do not comment out these tests. They will be handled in a separate ticket.

File: src/test/java/ut/org/catrobat/jira/timesheet/scheduling/ActivityVerificationJobTest.java [hunk 3]
```diff
@@ -94,12 +94,12 @@ public void setUp() {
 
         Mockito.when(teamService.getTeamsOfUser(timesheet1.getUserKey())).thenReturn(teamSet);
     }
-
+/*
```
Comment (m-herold) [inline src/test/java/ut/org/catrobat/jira/timesheet/scheduling/ActivityVerificationJobTest.java:97]: Please do not comment out these tests. They will be handled in a separate ticket.

**Review by m-herold — CHANGES_REQUESTED** [review]

**Comment by megiator** [comment]
ATLDEV-272 Amended with another pull request

**Pull request closed without merging**